### ISSUE TITLE
FC24 Live Editor

### ISSUE THREAD

**Issue by grasull** [xAranaktu/FC-24-Live-Editor#200]
Hi, i m seeing in the LE app that ig sees the TU as 1.0.89.6557.
Idk is that s the right TU or not BUT my live ediot closes the game after like 5-15 sec after entering it. 
Did all i ve could by watching 50+ responses on dif sites and nothing works.
How can i use it, i m actually down bad idk what to try more.

**Comment by zzlyz** [comment]
Yes this is the same for me i even tried changing injection times and that didnt fix anything i just want to play this with the cheat table :(((
